src/pybind/mgr/dashboard/frontend/src/app/core/error/error.component.scss

```diff
@@ -16,3 +16,7 @@ i {
 .fa-wrench {
   color: vv.$info;
 }
+
+.btn {
+  margin: 1em;
+}
```

Comment:
again, this should be avoided as much as we can and instead use the `ms-1` or similar.